### DISCUSSION TITLE
Windows构建源代码版失败，但无报错

### DISCUSSION THREAD

**Discussion by shaoxiu** [xfgryujk/blivechat#241] [提问] [answered]
前面的所有步骤一切正常，到了执行python3 main.py没有任何结果。反复clone多次都是这样。Python版本为3.13。

![PixPin_2024-10-15_16-16-14](https://github.com/user-attachments/assets/de6126c8-2444-42c3-81a1-d83e4c0643b1)

**Comment by xfgryujk** [comment, marked as answer]
把python3换成python？

**Reply by shaoxiu** [reply]
神医！我修改指令后提示SQLAlchemy版本过老，更新后run起来了。非常感谢帮助
